core/domain/exp_domain.py

```diff
@@ -354,7 +353,7 @@ def __init__(self, rule_type, inputs):
 
     def stringify_classified_rule(self):
         """Returns a string representation of a rule (for the stats log)."""
-        if self.rule_type == rule_domain.FUZZY_RULE_TYPE:
+        if self.rule_type == feconf.CLASSIFIER_RULE_TYPE:
```

Comment:
Prefer moving CLASSIFIER_RULE_TYPE to exp_domain. I think we want to move away from using feconf as a general bucket for constants. I think rule-related constants appropriately fit within exp_domain since nothing left in the backend really cares about rules.